web/src/components/molecules/Schema/FieldModal/FieldValidationProps/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+
+import { FieldType } from "../../types";
+
+export interface Props {
+  selectedType: FieldType | null;
+}
+
+const FieldValidationProps: React.FC<Props> = ({ selectedType }) => {
```

Comment:
Again, this isn't props but a component. Please think of a more meaningful name for this component.